meshroom/ui/qml/Application.qml

```diff
@@ -20,6 +20,23 @@ Page {
     property alias unsavedDialog: unsavedDialog
     property alias workspaceView: workspaceView
 
+    property var scenefile: _reconstruction ? _reconstruction.graph.filepath : "";
```

Comment:
```suggestion
    readonly property var scenefile: _reconstruction ? _reconstruction.graph.filepath : "";
```
Since there's no reason to ever edit this property from the QML side, we can probably set it as read-only.

Reply:
Nice catch !!, Thank you 🙂